### PR TITLE
Allow @return statements for function names containing (but not start…

### PR DESCRIPTION
**Pull request by bryanbraun**
…ing with) caps.

Fixes NikhilKalige/docblockr#17.

### CODE REVIEW

File: lib/docsparser.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ DocsParser.prototype.format_function = function(name, args, retval, options) {
 
 DocsParser.prototype.get_function_return_type = function(name, retval) {
     // returns None for no return type. False meaning unknown, or a string
-    if(name.search(/[A-Z]/) > -1)
+    if(name.search(/[A-Z]/) === 0)
```
Comment (robtarr) [inline lib/docsparser.js]: How about `if(name.match(/^[A-Z]/))`?

Comment (emflores) [inline lib/docsparser.js]: Or `if(/^[A-Z]/.test(name))` ;)

Comment (bryanbraun) [inline lib/docsparser.js]: ^^^ Agreed, I think this is a bit more clear @emflores. I tweaked the commit and pushed it back up.

**Comment by malko** [comment]
Merged in develop branch thanks to you

**Pull request closed without merging**